docs/reference/modules/cluster/disk_allocator.asciidoc

```diff
@@ -29,14 +29,21 @@ file or updated dynamically on a live cluster with the
     the node.
 
 `cluster.routing.allocation.disk.watermark.floodstage`::
-+
---
-    Controls the floodstage watermark. It defaults to 95%, meaning ES enforces a read-only
-    index block (`index.blocks.read_only_allow_delete`) on every index that has
-    one or more shards allocated on the node that has at least one disk exceeding the floodstage.
-    This is a last resort to prevent nodes from running out of disk space.
-    The index block must be released manually once there is enough disk space available
-    to allow indexing operations to continue.
+
+    Controls the flood stage watermark. It defaults to 95%, meaning ES enforces
+    a read-only index block (`index.blocks.read_only_allow_delete`) on every
+    index that has one or more shards allocated on the node that has at least
+    one disk exceeding the flood stage.  This is a last resort to prevent nodes
+    from running out of disk space.  The index block must be released manually
+    once there is enough disk space available to allow indexing operations to
+    continue.
+
+NOTE: You can not mix the usage of percentage values and byte values within these
+settings. Either all are set to percentage values, or all are set to byte
+values. This is so that we can we validate that the settings do not violate each
```

Comment:
I'd say something like "This is so that we can verify that the low watermark is less than the high watermark which is less than the flood stage threshold."